src/types.ts

```diff
@@ -72,3 +72,105 @@ export type Breadcrumbs = {
 };
 
 export type OrderedBreadcrumbs = Array<EnrichedCrumb>;
+
+export interface Passport {
+  data?: Record<string, any>;
```

Comment:
As discussed, it would be nice to eventually refactor this to a tighter definition (i.e. use the `Value` type introduced in #4 ) but not necessary for this change.

Reply:
Rebased and pulled this type in 👌

Reply:
This turns out to be a bit more complex than originally thought - multiple functions inside the OneApp class rely on being passed a certain type and not a `Value`.

In order to do this I'll need to get a `Passport().get()` function in place which I'll revisit as a discrete PR shortly in order to move forward on this sprint's tickets (send templates to email).